mlpipeline/encoder_service/test.py

```
@@ -0,0 +1,43 @@
+        raise FileNotFoundError(f"checkpoint not found: {ckpt_path}")
+    print("[OK] checkpoint exists")
+
+    ckpt = torch.load(ckpt_path, map_location="cpu")
+    state_dict = ckpt["state_dict"] if isinstance(ckpt, dict) and "state_dict" in ckpt else ckpt
+    print("[OK] loaded checkpoint")
+    print("[INFO] num params:", len(state_dict))
+    print("[INFO] first 5 keys:", list(state_dict.keys())[:5])
+
+    first_w = state_dict.get("mlp.0.weight", None)
+    if first_w is None:
+        raise KeyError("Cannot find 'mlp.0.weight' in state_dict. Please check printed keys above.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)